pupil-spa/src/app/login-failure/login-failure.component.html

```diff
@@ -10,7 +10,7 @@ <h1 tabindex="0">Sign-in error</h1>
         <strong class="visually-hidden">Warning</strong>
         <app-svg-warning></app-svg-warning>
         <strong>
-          Please tell your teacher
+          Please tell your teacher: {{errorMessage}}
```

Comment:
Should this be in this PR?

Reply:
Nope, I was trying to troubleshoot why the pupil spa wasn’t connecting to the API from inside it’s container. Good spot! Removed.